### PR TITLE
Add Polish keymap

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

🇵🇱

## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: quantum/keymap_extras/keymap_polish.h [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+#define PL_D    KC_D    // D
+#define PL_F    KC_F    // F
+#define PL_G    KC_G    // G
+#define PL_H    KC_H    // H
+#define PL_J    KC_J    // J
+#define PL_K    KC_K    // K
+#define PL_L    KC_L    // L
+#define PL_SCLN KC_SCLN // ;
+#define PL_QUOT KC_QUOT // '
+#define PL_BSLS KC_NUBS // (backslash)
```
Comment (vomindoraan) [inline quantum/keymap_extras/keymap_polish.h:75]: The Polish layout is based on US ASCII. I tested, both `KC_BSLS` and `KC_NUBS` work, but its more true to reality if `KC_BSLS` is used.
```suggestion
```
Moving to row 2.

File: quantum/keymap_extras/keymap_polish.h [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+#define PL_E    KC_E    // E
+#define PL_R    KC_R    // R
+#define PL_T    KC_T    // T
+#define PL_Y    KC_Y    // Y
+#define PL_U    KC_U    // U
+#define PL_I    KC_I    // I
+#define PL_O    KC_O    // O
+#define PL_P    KC_P    // P
+#define PL_LBRC KC_LBRC // [
+#define PL_RBRC KC_RBRC // ]
```
Comment (vomindoraan) [inline quantum/keymap_extras/keymap_polish.h:62]:
```suggestion
#define PL_RBRC KC_RBRC // ]
#define PL_BSLS KC_BSLS // (backslash)
```

File: quantum/keymap_extras/keymap_polish.h [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+// clang-format off
+
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ ` │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ \ │    │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
```
Comment (vomindoraan) [inline quantum/keymap_extras/keymap_polish.h:30]: As per the above comment.
```suggestion
 * │     │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │
 * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
 * │      │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │        │
 * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤
```

File: quantum/keymap_extras/keymap_polish.h [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+#define PL_HASH S(PL_3)    // !
+#define PL_DLR  S(PL_4)    // !
+#define PL_PERC S(PL_5)    // !
+#define PL_CIRC S(PL_6)    // !
+#define PL_AMPR S(PL_7)    // !
+#define PL_ASTR S(PL_8)    // !
+#define PL_LPRN S(PL_9)    // !
+#define PL_RPRN S(PL_0)    // !
+#define PL_UNDS S(PL_MINS) // !
+#define PL_PLUS S(PL_EQL)  // !
```
Comment (vomindoraan) [inline quantum/keymap_extras/keymap_polish.h:114]: Do you wish to imply that Polish people are _**loud**_? 😁
```suggestion
#define PL_AT   S(PL_2)    // @
#define PL_HASH S(PL_3)    // #
#define PL_DLR  S(PL_4)    // $
#define PL_PERC S(PL_5)    // %
#define PL_CIRC S(PL_6)    // ^
#define PL_AMPR S(PL_7)    // &
#define PL_ASTR S(PL_8)    // *
#define PL_LPRN S(PL_9)    // (
#define PL_RPRN S(PL_0)    // )
#define PL_UNDS S(PL_MINS) // _
#define PL_PLUS S(PL_EQL)  // +
```

**Comment by vomindoraan** [comment]
Could you bundle the following changes to the Slovak keymap together with this PR, please? I feel it doesn't deserve a PR of its own:
```diff
diff --git a/quantum/keymap_extras/keymap_slovak.h b/quantum/keymap_extras/keymap_slovak.h
index 7a02dcfdf..f0bd6ae51 100644
--- a/quantum/keymap_extras/keymap_slovak.h
+++ b/quantum/keymap_extras/keymap_slovak.h
@@ -36,10 +36,10 @@
 // Row 1
 #define SK_SCLN KC_GRV  // ;
 #define SK_PLUS KC_1    // +
-#define SK_LACU KC_2    // ľ
+#define SK_LCAR KC_2    // ľ
 #define SK_SCAR KC_3    // š
 #define SK_CCAR KC_4    // č
-#define SK_TACU KC_5    // ť
+#define SK_TCAR KC_5    // ť
 #define SK_ZCAR KC_6    // ž
 #define SK_YACU KC_7    // ý
 #define SK_AACU KC_8    // á
@@ -141,10 +141,10 @@
  */
 // Row 1
 #define SK_TILD ALGR(SK_PLUS) // ~
-#define SK_CIRC ALGR(SK_LACU) // ^ (dead)
+#define SK_CIRC ALGR(SK_LCAR) // ^ (dead)
 #define SK_BREV ALGR(SK_SCAR) // ˘ (dead)
 #define SK_RNGA ALGR(SK_CCAR) // ° (dead)
-#define SK_OGON ALGR(SK_TACU) // ˛ (dead)
+#define SK_OGON ALGR(SK_TCAR) // ˛ (dead)
 #define SK_GRV  ALGR(SK_ZCAR) // `
 #define SK_DOTA ALGR(SK_YACU) // ˙ (dead)
 #define SK_DACU ALGR(SK_EACU) // ˝ (dead)
```
I overlooked this when I was reviewing #8561. Even though the lowercase `ľ` and `ť` are written with what looks like an acute accent, it's actually a caron (_mäkčeň_).

**Comment by fauxpark** [comment]
Good spot

**Pull request merged**